content/g-sgov/g-sgov-sportoviště/g-sgov-sportoviště-diagram-MaNe.ttl

```diff
@@ -25,5 +25,5 @@ g-sgov-sportoviště:diagram
                 "g-sgov-sportoviště-pojem" ;
         <http://purl.org/vocab/vann/preferredNamespaceUri>
                 "https://slovník.gov.cz/generický/sportoviště/pojem/" ;
-        owl:imports     g-sgov-sportoviště:glosář ;
+        owl:imports     g-sgov-sportoviště:glosář , <https://slovník.gov.cz/generický/veřejná-místa/diagram> ;
```

Comment:
Proč importujeme i diagramy navzájem ?!?

Reply:
Moc to smysl nedava, no exportuje to takto modelio.  Kdyz jsem implementoval diagramy, tak mne to zrejme nenapadlo testovat to na importech, jelikoz jsme je prakticky nepouzivali ... Nebo to tam bylo zavleceno pozdeji ?! Nevim no diagramy bude potreba zrejme revidovat.